engine/clients/redis/helper.py

```diff
@@ -0,0 +1,9 @@
+MIN_LAT, MAX_LAT = -85.05112878, 85.05112878
+
+
+def epsg_4326_to_900913(lon, lat):
```

Comment:
maybe we can rename it to something more human-readable and left details in comment or docstring?

Reply:
That's right and done :)